receiver/hostmetricsreceiver/internal/scraper/cpuscraper/factory_test.go

```diff
@@ -25,5 +25,4 @@ func TestCreateMetricsScraper(t *testing.T) {
 
 	assert.NoError(t, err)
 	assert.NotNil(t, scraper)
-	assert.Equal(t, scraperType.String(), scraper.ID().String())
```

Comment:
@dmitryax @bogdandrutu is this an expected outcome of the scraper changes? The godocs say to use `AddScraperWithType` instead of `scraper.ID`, but these hostmetric tests are testing `factory.CreateMetricsScraper` which returns a `scraperhelper.Scraper` and are not calling `NewScraperControllerReceiver` so `AddScraperWithType` isn't an option.

Reply:
I'm gonna merge to continue with the release because:
1. these are tests and can be changed whenever
2. the API that affected these tests is following the deprecation cycle and therefore can be reverted easily in a future release if needed.

Reply:
Agree we can merge, can we have an issue about it to track it?